cli/schedule.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+from datetime import datetime
+
+import click
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+from indexer.schedule_jobs.aggregates_jobs import aggregates_yesterday_job, parse_crontab
+
+
+@click.command(context_settings=dict(help_option_names=["-h", "--help"]))
+@click.option(
+    "-cn",
+    "--chain-name",
+    default=None,
+    show_default=True,
+    type=str,
+    help="The chain name of the chain to aggregate data for",
+    envvar="CHAIN_NAME",
+)
+@click.option(
+    "-jn",
+    "--job-name",
+    default=None,
+    show_default=True,
+    type=str,
+    help="Job list to aggregate data for",
+    envvar="JOB_NAME",
+)
+@click.option(
+    "-pg",
+    "--postgres-url",
+    type=str,
+    required=True,
+    envvar="POSTGRES_URL",
+    help="The required postgres connection url." "e.g. postgresql+psycopg2://postgres:admin@127.0.0.1:5432/ethereum",
+)
+@click.option(
```

Comment:
I suggest not to add this. You can assume you have token price table in postgres-url

Reply:
Sure